[DOC] Update Tempo and Pyroscope data source doc (#91704)
grafana/grafana: docs/sources/shared/datasources/tempo-traces-to-profiles.md

1.  Select a Pyroscope data source in the **Data source** drop-down.
1.  Optional: Choose any tags to use in the query. If left blank, the default values of `service.name` and `service.namespace` are used.

>     These tags can be used in the custom query with `${__tags}` variable. This variable interpolates the mapped tags as list in an appropriate syntax for the data source. Only the tags that were present in the span are included. Tags that aren't present are omitted.
> 
>     You can also configure a name for the tag. This is useful in cases where the tag has dots in the name and the target data source doesn't allow using dots in labels. For example, you can remap `service.name` to `service_name`. If you don’t map any tags here, you can still use any tag in the query, for example: `method="${__span.tags.method}"`. You can learn more about custom query variables [here](/docs/grafana/<GRAFANA_VERSION>/datasources/tempo/configure-tempo-data-source/#custom-query-variables).

Author: ```suggestion
    These tags can be used in the custom query with `${__tags}` variable. This variable interpolates the mapped tags as list in an appropriate syntax for the data source. Only tags present in the span are included. Tags that aren't present are omitted.

    You can also configure a name for the tag. Tag names are useful where the tag has dots in the name and the target data source doesn't allow using dots in labels. For example, you can remap `service.name` to `service_name`. If you don’t map any tags here, you can still use any tag in the query, for example: `method="${__span.tags.method}"`. Learn more about [custom query variables](/docs/grafana/<GRAFANA_VERSION>/datasources/tempo/configure-tempo-data-source/#custom-query-variables).
```

1.  Select one or more profile types to use in the query. Select the drop-down and choose options from the menu.
1.  Switch on **Use custom query** to enter a custom query.
1.  Specify a custom query to be used to query profile data. You can use various variables to make that query relevant for current span. The link shows only if all the variables are interpolated with non-empty values to prevent creating an invalid query. You can interpolate the configured tags using the `$__tags` keyword.
1.  Select **Save and Test**.

## Configuration options

> The following table describes options for configuring your Trace to profiles settings:

Reviewer: ```suggestion
The following table describes options for configuring your **Trace to profiles** settings:
```

| **Tags**             | Defines the tags to use in the profile query. Default: `cluster`, `hostname`, `namespace`, `pod`, `service.name`, `service.namespace`. You can change the tag name for example to remove dots from the name if they're not allowed in the target data source. For example, map `http.status` to `http_status`. |
| **Profile type**     | Defines the profile type that used in the query.                                                                                                                                                                                                                                                               |
| **Use custom query** | Toggles use of custom query with interpolation.                                                                                                                                                                                                                                                                |

> | **Query**            | Input to write custom query. Use variable interpolation to customize it with variables from span.                                                                                                                                                                                                              |

Reviewer: Don't bold these UI elements because they're the only thing in the table cell.

Author: Even though these are UI elements?

Author: Fixed.

> | **Data source**      | Defines the target data source. You can select a Pyroscope \[profiling\] data source.                                                                                                                                                                                                                          |
> | **Tags**             | Defines the tags to use in the profile query. Default: `cluster`, `hostname`, `namespace`, `pod`, `service.name`, `service.namespace`. You can change the tag name for example to remove dots from the name if they're not allowed in the target data source. For example, map `http.status` to `http_status`. |
> | **Profile type**     | Defines the profile type that used in the query.                                                                                                                                                                                                                                                               |
> | **Use custom query** | Toggles use of custom query with interpolation.                                                                                                                                                                                                                                                                |
> | **Query**            | Input to write custom query. Use variable interpolation to customize it with variables from span.                                                                                                                                                                                                              |

Author: ```suggestion
| Data source      | Defines the target data source. You can select a Pyroscope \[profiling\] data source.                                                                                                                                                                                                                          |
| Tags             | Defines the tags to use in the profile query. Default: `cluster`, `hostname`, `namespace`, `pod`, `service.name`, `service.namespace`. You can change the tag name for example to remove dots from the name if they're not allowed in the target data source. For example, map `http.status` to `http_status`. |
| Profile type     | Defines the profile type that used in the query.                                                                                                                                                                                                                                                               |
| Use custom query | Toggles use of custom query with interpolation.                                                                                                                                                                                                                                                                |
| Query            | Input to write custom query. Use variable interpolation to customize it with variables from span.                                                                                                                                                                                                              |
```
